Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM node:10.10.0
 
-RUN apt-get update && apt-get upgrade -y
+RUN echo "deb [check-valid-until=no] http://archive.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
```

Comment:
Can you add in a quick comment about why you are doing this? Something like:
"Fixing EOL jessie update repos (issue #1599)"

Reply:
I just threw the comment right above, is that alright? Or did you want it inline?

Reply:
Above is perfect! I'll just be super nitpicky for your first PR or two so that we can get on the same page about style: I would suggest for comments to start with a space and a capital letter and end with a comment. So it would look like:
`# Fixing EOL jessie update repos (issue #1599).`

I need to write up a style guide to send to new students :grin:

Reply:
Yes absolutely, I appreciate all the help.